common/src/test/java/org/conscrypt/javax/crypto/CipherTest.java

```diff
@@ -4014,6 +4031,13 @@ private void checkCipher(CipherTestParam p, String provider) throws Exception {
         }
     }
 
+    // SunJCE has known issues between 17 and 21
+    private boolean isBuggyProvider(String providerName) {
+        return providerName.equals("SunJCE")
+                && TestUtils.isJavaVersion(17)
+                && !TestUtils.isJavaVersion(21);
```

Comment:
Is `isJavaVersion` actually `isJavaVersionAtLeast`?

Reply:
Yeah, I think that dates back to when they only cared about 7 and 8.   I'll rename it  in another CL because there's a few other things about Java version which are a bit icky.